### PR TITLE
[DJFS-112] Develop a List View for Customer Orders

### PR DESCRIPTION
**Pull request by ainobs**
Screenshot (120)

## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/victorAienobe/phpDatabase/listView.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<?php
+
+    include_once('connection.php');
+    $id = $_GET['id'];
+
+    class View extends  Connect {
+        public function list($id) {
+            try {
+                //$sql = "SELECT * FROM orders";
+               $sql= "SELECT orders.product_name, orders.order_unit_price,orders.order_quantity,
```
[the remark this answers is not in the document]
Comment (ainobs) [reply submissions/victorAienobe/phpDatabase/listView.php:10]: I made use of the JOIN clause, the first line is commented out. That wasn't the query I used.

File: submissions/victorAienobe/phpDatabase/customerDatabase.sql [hunk 2]
```diff
@@ -11,4 +11,23 @@ INSERT INTO customer_details (full_name, email_address, created_at) VALUES
 ('Micheal Scofield', 'scofield@gmail.com', now()),
 ('Lincoln Burrows', 'burrows@gmail.com', now()),
 ('Theodore Bagwell', 'bagwell@gmail.com', now()),
-('John Abruzzi', 'abruzzi@gmail.com', now());
\ No newline at end of file
+('John Abruzzi', 'abruzzi@gmail.com', now());
+
+CREATE TABLE orders(
```
[the remark this answers is not in the document]
Comment (ainobs) [reply submissions/victorAienobe/phpDatabase/customerDatabase.sql:16]: I have just two tables..
One for the previous task, and another for this task to accommodate the customer orders. I don't understand what you mean by changing DB structure

[the remark this answers is not in the document]
Comment (adaugochi) [reply submissions/victorAienobe/phpDatabase/customerDatabase.sql:16]: @ainobs You are not expected to create a new table. Kindly work with the existing tables you have on e-commerce database.

File: submissions/victorAienobe/phpDatabase/listView.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (adaugochi) [reply submissions/victorAienobe/phpDatabase/listView.php:10]: Hint:

The query you are to write here will touch about 4 tables. `orders`, `order_items`, `customer_addresses` and `customers` table. These table already have relationship existing in them

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/victorAienobe/phpDatabase/customerDatabase.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (ainobs) [reply submissions/victorAienobe/phpDatabase/customerDatabase.sql:16]: okay i understand you now...
i thought we were meant to create a new  database for the task, which i did...

File: submissions/victorAienobe/phpDatabase/listView.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (ainobs) [reply submissions/victorAienobe/phpDatabase/listView.php]: thanks, i've made the correction
kindly review

**Review by adaugochi — APPROVED** [review]

**Pull request merged**